src/zeit/content/volume/volume.py

```diff
@@ -170,40 +170,42 @@ def _is_valid_cover_id_and_product_id(self, cover_id, product_id):
         product_ids = [prod.id for prod in self._all_products]
         return cover_id in cover_ids and product_id in product_ids
 
-    def all_content_via_solr(self, additional_query_contstraints=None):
+    def all_content_via_search(self, additional_query_contstraints=None):
         """
-        Get all content for this volume via Solr.
-        If u pass a list of additional query strings, they will be added as
-        an AND-operand to the query field.
+        Get all content for this volume via ES.
+        If u pass a list of additional query clauses, they will be added as
+        an AND-operand to the query.
         """
         if not additional_query_contstraints:
             additional_query_contstraints = []
-        Q = zeit.solr.query
-        solr = zope.component.getUtility(zeit.solr.interfaces.ISolr)
-        query = Q.and_(
-            Q.not_(Q.field('uniqueId', self.uniqueId)),
-            Q.or_(*[Q.field('product_id', p.id) for p in
-                    self._all_products]),
-            Q.field_raw('year', self.year),
-            Q.field_raw('volume', self.volume),
-            *additional_query_contstraints
-        )
-        result = solr.search(query, fl='uniqueId', rows=1000)
+        elastic = zope.component.getUtility(zeit.find.interfaces.ICMSSearch)
```

Comment:
Nur, um sicher zu gehen: hier benutzen wir das utility aus `zeit.find`, um auch unpublished content zu bekommen (um den dann veröffentlichen zu können), richtig?  Das *all* in `all_content_via_search` ist ja schon Indiz, aber damit ich's richtig verstehe... 😊

Reply:
Ah, ja...  wie immer: erst zu Ende lesen, dann fragen! 😂